src/instana/collector/base.py

```diff
@@ -150,12 +150,6 @@ def background_report(self):
 
         self.prepare_and_report_data()
 
-        if self.thread_shutdown.is_set():
-            logger.debug(
-                "Thread shutdown signal is active: Shutting down reporting thread"
-            )
-            return False
-
```

Comment:
Any specific reason to remove this?

Reply:
It was removed by a mistake, re-added it